### PR TITLE
[-] BO : fixed permissions display in 1.7

### PR DESCRIPTION
**Pull request by mickaelandrieu**
## Description

permissions
## Steps to Test this Fix

Access Permissions Tab on Configure > Team page
## Forge Ticket (optional)

"Fixes ticket http://forge.prestashop.com/browse/BOOM-14"

### CODE REVIEW

**Comment by xBorderie** [comment]
@AlexEven Could you see about the wording here? Thanks!

File: controllers/admin/AdminAccessController.php [hunk 1]
```diff
@@ -27,6 +27,9 @@
 /**
  * @property Profile $object
  */
+
+use Symfony\Component\VarDumper\VarDumper;
```
Comment (julienbourdeau) [inline controllers/admin/AdminAccessController.php]: :dart:

**Comment by AlexEven** [comment]
Seems consistent with what we have in the new menu, looks good to me!

**Comment by mickaelandrieu** [comment]
@AlexEven @xBorderie all information from permission list come from the tabs data :)

**Pull request merged**